thrift/lib/cpp2/Makefile.am

```diff
@@ -173,7 +173,7 @@ libthriftprotocol_la_SOURCES = Version.cpp \
                                protocol/SimpleJSONProtocol.cpp \
                                protocol/VirtualProtocol.cpp
 
-libthriftprotocol_la_LIBADD = $(FOLLY_LIBS) $(BOOST_SYSTEM_LIB)
+libthriftprotocol_la_LIBADD = $(FOLLY_LIBS) $(BOOST_SYSTEM_LIB) -ldouble-conversion
```

Comment:
I'd suggest doing that as a follow up diff; fixing it in folly won't break this, plus I just clicked a button to get this landed :-)